leakcanary-android/src/main/java/com/squareup/leakcanary/internal/LeakCanaryInternals.java

```diff
@@ -107,6 +107,14 @@ public static void setEnabled(Context context, final Class<?> componentClass,
     });
   }
 
+  /** Based on http://stackoverflow.com/a/3758880. */
+  public static String humanReadableByteCount(long bytes) {
```

Comment:
Maybe. But it will be used in context like "Activity has leaked 51.4 KiB", so reader might get it right away.